spec/features/register_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'spec_helper'
+
+feature 'user registration', {vcr: true, js: true} do
+
+  background do
+    visit register_path
+  end
+
+  scenario 'both, user data valid and credit card data valid' do
+    fill_in 'Email', with: 'user@mailinator.com'
+    fill_in 'Password', with: '123'
+    fill_in 'Password confirmation', with: '123'
+    fill_in 'Full name', with: 'User Mailinator'
+    fill_in_credit_card_data '4242424242424242'
```

Comment:
You could refactor this part:

```ruby
    fill_in 'Email', with: 'user@mailinator.com'
    fill_in 'Password', with: '123'
    fill_in 'Password confirmation', with: '123'
    fill_in 'Full name', with: 'User Mailinator'
    fill_in_credit_card_data '4242424242424242'
```
To some sort of a method. In this spec it does not work with `fill_in_credit_card_data` - as that is already another level of abstraction.

It asks to be refactored to something like "fill_in_user_details".